frontend-react/src/pages/submissions/SubmissionsTable.tsx

```diff
@@ -1,17 +1,78 @@
-import { useContext } from "react";
+import { useContext, useState } from "react";
 import moment from "moment";
 import { useResource } from "rest-hooks";
+import {
+    Button,
+    IconNavigateBefore,
+    IconNavigateNext,
+} from "@trussworks/react-uswds";
 
 import SubmissionsResource from "../../resources/SubmissionsResource";
 import { GlobalContext } from "../../components/GlobalContextProvider";
 
 function SubmissionsTable() {
     // this component will refresh when global context changes (e.g. organization changes)
     const globalState = useContext(GlobalContext);
+
+    // state of pagination
+    const [paginationCursor, setPaginationCursor] = useState("");
+    const [paginationSort, setPaginationSort] = useState("DESC");
+    const [paginationPageSize] = useState(10);
+
     const submissions: SubmissionsResource[] = useResource(
         SubmissionsResource.list(),
-        { organization: globalState.state.organization }
+        {
+            organization: globalState.state.organization,
+            cursor: paginationCursor,
+            pageSize: paginationPageSize,
+            sort: paginationSort,
+        }
     );
+
+    const sortedSubmissions = () => {
+        if (paginationSort === "ASC") {
+            // sort by createdAt DESC
+            // because when the paginationSort is ASC, the results from the server are reversed
+            submissions.sort(
+                (a, b) =>
+                    Date.parse(b.createdAt?.toString() || "") -
```

Comment:
Date.parse("") returns NaN. I guess this field should never be blank. You could make it "1/1/2020" and put it in a local const if you wanted. Purely optional.